tests/integration/test_complete_run.cfg

```diff
@@ -33,6 +33,12 @@ years = "1850:1854:2",
   input_files = "eam.h0"
   input_subdir = "archive/atm/hist"
 
+  [[ atm_monthly_180x360_aave_ilamb ]]
```

Comment:
could this be [[ atm_monthly_180x360_aave ]]? I think the logic is if ts_fmt=cmip, both nco convention time series and cmip time series are generated.
And this sub session can be both a dependency of both e3sm_diags and ilamb.

Reply:
Thanks, it must have been some issue in debugging that made me think it required a separate subtask. It looks like it works fine using the same subtask.